style.css

```diff
@@ -1,18 +1,19 @@
-h1{
-    font-size: 4rem;
-    margin-bottom: 0;
-    text-align: center;
-    padding: 20px 500px;
-    background-color: red;
+h1 {
+  font-size: 4rem;
+  margin-bottom: 0;
+  text-align: center;
+  padding: 20px 500px;
+  background-color: red;
 }
 
-img{
-    display: block;
+img {
+  font-size: 4rem;
```

Comment:
I think it would be better not to give an image a font size because that does not apply to images.

Reply:
- [ ] Great job here 👏 you were very specific and to the point ✔. Just one more tip try to use emojis as part of your comments as they make the experience more human-friendly 😊